commands/standalone.js

```diff
@@ -10,10 +17,34 @@ process.env.COZY_FIELDS = JSON.stringify({
 const config = require('../helpers/init-konnector-config')()
 process.env.COZY_URL = config.COZY_URL
 
-const filename = process.argv[2] || 'index.js'
+const filename = program.args[0] || 'index.js'
```

Comment:
This could be done via commander ?

Reply:
I can't find a way to do it with commander configuration

Reply:
const program = require('commander')

let cmd, env
program
  .arguments('<cmd> [env]')
  .action((_cmd, _env) => {
    cmd = _cmd
    env = _env
  })

program.parse(process.argv)
if (!cmd) {
    console.log('index:', program.usage())
    process.exit(1)
}

console.log(cmd, env)

Reply:
Je viens de découvrir qu'il y a un port de la librairie argparse en JS. Personnellement je préfère leur API à commander ou yargs.

Reply:
J'étais parti sur minimist (j'aime bien le minimalisme avant de voir qu'il y avait commander déjà embarqué.

Je dois dire que je suis pas un grand fan de argparse.

Vu que konitor utilise déjà yargs, on peut peut-être partir dessus. Yargs est assez gros mais ce ne sera pas un problème une fois qu'on aura séparé cli et libs.